models/product/src/product-data/transformers.ts

```diff
@@ -59,7 +59,7 @@ const transformers = {
     },
   }),
   graphql: Transformer<TProductData, TProductDataGraphql>('graphql', {
-    buildFields: ['categories'],
+    buildFields: ['categories', 'masterVariant'],
```

Comment:
We need the graphql transformer to build the `masterVariant` because otherwise consuming the generated data will fail.

Reply:
Nice catch 👍